fwdis:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  front_right_steering_joint:
    type: velocity_controllers/JointPositionController
    joint: front_right_steering_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  front_left_steering_joint:
    type: velocity_controllers/JointPositionController
    joint: front_left_steering_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  rear_right_steering_joint:
    type: velocity_controllers/JointPositionController
    joint: rear_right_steering_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  rear_left_steering_joint:
    type: velocity_controllers/JointPositionController
    joint: rear_left_steering_joint
    pid: {p: 5.0, i: 0.0, d: 0.0}

  front_right_wheel_joint:
    type: velocity_controllers/JointVelocityController
    joint: front_right_wheel_joint

  front_left_wheel_joint:
    type: velocity_controllers/JointVelocityController
    joint: front_left_wheel_joint

  rear_right_wheel_joint:
    type: velocity_controllers/JointVelocityController
    joint: rear_right_wheel_joint

  rear_left_wheel_joint:
    type: velocity_controllers/JointVelocityController
    joint: rear_left_wheel_joint
